docs/configure/images-and-assets.md

```diff
@@ -137,3 +135,16 @@ In this case, you need to have all your images and media files with relative pat
 If you load static content via importing, this is automatic, and you do not have to do anything.
 
 Suppose you are serving assets in a [static directory](#serving-static-files-via-storybook-configuration) along with your Storybook. In that case, you need to use relative paths to load images or use the base element.
+
+### Referencing Fonts in Stories
+
+You've placed the fonts in your static folder, now what? 
```

Comment:
😄 Love it. It's not quite the tone of the rest of ours docs, though. What about this?

```suggestion
After configuring Storybook to server assets from your static folder, you can reference those assets in Storybook. For example, you can reference and apply a custom font in your stories.
```